TA_Scheduler/utilities/AccountUtil.py

```diff
@@ -141,3 +141,19 @@ def getAccountByUsername(username: str) -> Optional[Account]:
             return None
         except IndexError:
             return None
+
+    def updateAccountInfo(
+        id: int, first: str, last: str, email: str, address: str, phone: int
+    ):
+        if account := AccountUtil.getAccountByID(id):
+            if first is not None:
+                account.user.first_name = first
+            if last is not None:
+                account.user.last_name = last
+            if email is not None:
+                account.user.email = email
```

Comment:
if any of these are `None` We should raise a `TypeError` with a message since they are required fields

Reply:
I intended this utility method to be more flexible than what is required for the particular use of it in creating an account. 
For example, it could be used in the future on an edit page. The user would already have the required fields filled, so it doesn't need to update those ones.

Reply:
Ok that sounds good I like that.

I think I would still recommend throwing an exception for `id` being `None`

The only thing I would recommend then is giving the default value of `None` to the arguments, that way when you are only updating parts of an account, you only have to call the function with what you want to update, and leave the rest blank.

Ex 
`updateAccountInfo(id: int = None, first: str = None, last: str =None):`

Then a call to update the first name would just be 
`updateAccountInfo(id,"Jones")`

Reply:
Okay, made those updates and pushed to my branch.

Reply:
Looks good.
Probably the last suggestion is to return False if no account is found with that ID to confirm that changes were made

Reply:
Looks great! 
I gata work on how I go about pr reviews cuz I feel mean coming back with the little changes one at a time.
I think in the future I will suggest what I think the interface should look like (args/returns/exceptions) which would have covered everything.
:) thx for being patient with me!